python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -245,7 +247,24 @@ class FgcmMakeLutConfig(pipeBase.PipelineTaskConfig,
              "which allows replacement (or time-variable) filters to be "
              "properly cross-calibrated."),
         dtype=str,
-        default=None,
+        default=[],
+        deprecated=("This field is no longer used, and has been deprecated by "
+                    "DM-28088.  It will be removed after v22.  Use "
+                    "stdPhysicalFilterMap instead.")
+    )
+    physicalFilters = pexConfig.ListField(
+        doc="List of physicalFilter labels to generate look-up table.",
+        dtype=str,
+        default=[],
+    )
+    stdPhysicalFilterOverrideMap = pexConfig.DictField(
+        doc=("Override mapping from physical filter labels to 'standard' physical "
```

Comment:
As I think about this, is "standard" really the right word for this? I'm trying to figure out what is "standard" about `HSC-R2` vs. `HSC-R`. Does "standard" mean "standardized" or "normalized"?

Reply:
I'm using "standard" as in "standard bandpass".

Reply:
But that's what "band label" is for, not "physical label"? I think this is why this config confuses me: physical label is meant to be unique to a given piece of glass. I could see standardizing across close-but-not-quite physical filters, but that's not really a "standard bandpass": you wouldn't do that across `HSC-R` and `DECAM r`, would you?

Reply:
A band is a label.  A standard bandpass is a `T(lambda)`.  You define your standard bandpass (which is filter + optics + sensors + standard atmosphere), and calibrate everything relative to that.  You would either put `HSC-R` on the `DECAM r` system or vice versa.  This allows you to specify what you want your standard bandpass `T(lambda)` to be based on.  So this is using "standard filter" in the same way as "standard atmosphere", it's the throughput the standard is based on.

Reply:
Ok, thanks for the clarification. I'm trying to think of how to make that more clear in the docstring.